slugger_test.go

```diff
@@ -13,15 +13,20 @@ var _ = Suite(&SluggerSuite{})
 
 func (s *SluggerSuite) TestSlugCreationNormalText(c *C) {
 	title := "This is a fake title"
-	c.Assert(SlugifyTitle(title), Equals, "this-is-a-fake-title")
+	c.Assert(SlugifyTitle(title, "-"), Equals, "this-is-a-fake-title")
 }
 
 func (s *SluggerSuite) TestSlugCreationWithSpecialChars(c *C) {
 	secondTitle := "This &&& *** is another title"
-	c.Assert(SlugifyTitle(secondTitle), Equals, "this-is-another-title")
+	c.Assert(SlugifyTitle(secondTitle, "_"), Equals, "this_is_another_title")
 }
 
 func (s *SluggerSuite) TestSlugCreationWithMultipleSpaces(c *C) {
 	title := "This is a                    fake title"
-	c.Assert(SlugifyTitle(title), Equals, "this-is-a-fake-title")
+	c.Assert(SlugifyTitle(title, "+"), Equals, "this+is+a+fake+title")
+}
+
+func (s *SluggerSuite) TestSlugCreationWithSpecialCharAtTheEnd(c *C) {
+	title := "This is a fake (title)"
+	c.Assert(SlugifyTitle(title, "+"), Equals, "this+is+a+fake+title")
 }
```

Comment:
Added this test as my modifications to the logic of the `SlugifyTitle` class introduced a bug in some edge case